### ISSUE TITLE
Feature request: Storage blocks

### ISSUE THREAD

**Issue by Sranshaft** [grompe/BTWTweak#4]
As the title suggests, would there be any chance to add more storage blocks; ex., bone blocks, rotten flesh blocks, back-porting slime and coal blocks, etc.

**Comment by grompe** [comment]
Definitely would like that.

Requirements:
- extended block IDs
- block textures
- slime block logic

**Comment by Sranshaft** [comment]
Slime block logic isn't necessarily needed; I didn't like the way Mojang implemented it anyway. Just having a way to store all of the slime in a more compact way would definitely be a welcomed addition though.

If you need textures, I'd be happy to help you out there.

I guess block IDs are the biggest issue. Would it be necessary to have a separate ID for each new block or could you implement / extend a base storage block (GPEBlockStorage) and use metadata to differentiate the individual blocks? That would allow up to 16 new blocks if I'm not mistaken. Similar to what FlowerChild did with FCBlockAestheticOpaque.

**Comment by grompe** [comment]
Yes, I don't like the bouncy slime block either, the way it interacts with pistons is more of an interest...

I also like the idea of encouraging the use of the Hardcore Packing feature.

The list so far:

- Prerequisite: extended block IDs, like Deco Add-On does it
- Craft 9x on a bench or with Hardcore Packing the following blocks:
  - Separate soap blocks into a new separate ID and make it slippery as ice
  - Coal block (ensure texture is different from black stone)
  - Charcoal block (same as above)
  - Nethercoal block
  - Backport slime blocks, with different logic: sticky rather than bouncy: allows walking on surface with slowdown, but if jumped on, works like cobweb but suffocating
  - Sugar block
  - Silk block (maybe?)
  - Gunpowder and nitre blocks (maybe?)
  - Hay block (if Deco is not present, use same ID)
  - Diamond ingot block (if Deco is not present, use same ID)
- Make the following craftable only with Hardcore Packing:
  - Meat block (from 9 of any meat acceptable for sandwiches)
    - To prevent it from becoming a hyper-efficient food storage,
      allow cutting into 9 meat "ingots" only by saw
  - Flesh block
  - Bone block
  - Furniture block (9x sawdust, no crafting back except cutting by saw)

**Comment by Sranshaft** [comment]
I like what you've come up with and definitely think hardcore packing could use more uses. 

The one thing I would change is the recipe for the sawdust block. I would make it one glue surrounded by eight sawdust. This would give another use for glue as there isn't enough uses for it at the moment and would make more sense, in a Minecraft shirt of way. It would also make cooking/ smelting

**Comment by grompe** [comment]
How it looks so far:

![](http://i.imgur.com/lS3fqvL.png)
